### PR TITLE
enh: add oasis joint-fusion label atlas

### PR DESCRIPTION
**Pull request by mgxd**

### CODE REVIEW

File: niworkflows/data/getters.py [hunk 1]
```diff
@@ -22,7 +22,8 @@
     'mni152_nlin_sym_las': ('57fa7fc89ad5a101e635eeef', '9c4c0cad2a2e99d6799f01abf4107f5a'),
     'mni152_nlin_sym_ras': ('57fa7fd09ad5a101df35eed0', '65d64ad5a980da86e7d07d95b3ed2ccb'),
     'mni_icbm152_linear': ('580705eb594d9001ed622649', '72be639e92532def7caad75cb4058e83'),
-    'mni_icbm152_nlin_asym_09c': ('580705089ad5a101f17944a9', '002f9bf24dc5c32de50c03f01fa539ec')
+    'mni_icbm152_nlin_asym_09c': ('580705089ad5a101f17944a9', '002f9bf24dc5c32de50c03f01fa539ec'),
+    'oasis_dtk31_mni152': ('5a95995e91b689000ea221fe', '6eb144c5e3f5a375f23ffc3dc2d1cc2f')
```
Comment (effigies) [inline niworkflows/data/getters.py]: Can you add a comma at the end here, so that future additions only affect the one line?

**Comment by mgxd** [comment]
looks like a caching problem, perhaps needs to be restarted?

**Comment by effigies** [comment]
Restarted, but not sure if that invalidates the cache. "Re-run without cache" doesn't seem to exist anymore...

File: niworkflows/data/getters.py [hunk 2]
```diff
@@ -22,7 +22,8 @@
     'mni152_nlin_sym_las': ('57fa7fc89ad5a101e635eeef', '9c4c0cad2a2e99d6799f01abf4107f5a'),
     'mni152_nlin_sym_ras': ('57fa7fd09ad5a101df35eed0', '65d64ad5a980da86e7d07d95b3ed2ccb'),
     'mni_icbm152_linear': ('580705eb594d9001ed622649', '72be639e92532def7caad75cb4058e83'),
-    'mni_icbm152_nlin_asym_09c': ('580705089ad5a101f17944a9', '002f9bf24dc5c32de50c03f01fa539ec')
+    'mni_icbm152_nlin_asym_09c': ('580705089ad5a101f17944a9', '002f9bf24dc5c32de50c03f01fa539ec'),
+    'oasis_dtk31_mni152': ('5a95995e91b689000ea221fe', '6eb144c5e3f5a375f23ffc3dc2d1cc2f'),
```
Comment (effigies) [inline niworkflows/data/getters.py]: I think you actually want `dkt31`, assuming it refers to "Desikan-Killiany-Tourville".

Comment (mgxd) [reply niworkflows/data/getters.py]: looks like i fat fingered that, good catch

**Comment by effigies** [comment]
LGTM. @oesteban any concerns?

File: niworkflows/data/getters.py [hunk 3]
```diff
@@ -22,7 +22,8 @@
     'mni152_nlin_sym_las': ('57fa7fc89ad5a101e635eeef', '9c4c0cad2a2e99d6799f01abf4107f5a'),
     'mni152_nlin_sym_ras': ('57fa7fd09ad5a101df35eed0', '65d64ad5a980da86e7d07d95b3ed2ccb'),
     'mni_icbm152_linear': ('580705eb594d9001ed622649', '72be639e92532def7caad75cb4058e83'),
-    'mni_icbm152_nlin_asym_09c': ('580705089ad5a101f17944a9', '002f9bf24dc5c32de50c03f01fa539ec')
+    'mni_icbm152_nlin_asym_09c': ('580705089ad5a101f17944a9', '002f9bf24dc5c32de50c03f01fa539ec'),
+    'oasis_dkt31_mni152': ('5a95cd9591b689000ca5b6ab', 'e0bea438b9645c9e7c333a6edd859715'),
```
Comment (satra) [inline niworkflows/data/getters.py]: @oesteban, @mgxd - was this regenerated by registering the OASIS template to the mni asym template? or simply used as being in MNI space. if that latter, i would highly recommend running the registration and then moving the atlas over.

Comment (mgxd) [reply niworkflows/data/getters.py]: this was generated by registering the oasis template brainmask to the 2mm mni-asym template, and then applying the transform to the label atlas

**Pull request merged**